cli/pcluster/utils.py

```diff
@@ -269,8 +269,9 @@ def get_instance_vcpus(region, instance_type):
     or the pricing file cannot be retrieved/parsed
     """
     try:
-        instances = _get_json_from_s3(region, "instances/instances.json")
-        vcpus = int(instances[instance_type]["vcpus"])
+        instance_type_info = get_instance_type(instance_type)
+        vcpus_info = instance_type_info.get("VCpuInfo")
+        vcpus = vcpus_info.get("DefaultVCpus")
     except (KeyError, ValueError, ClientError):
```

Comment:
I think the ValueError is not more valid.

Reply:
Fixed